services/bower/bower-version.tester.js

```diff
@@ -44,10 +49,15 @@ t.create('Version label should be `no releases` if no stable version')
   .expectBadge({ label: 'bower', message: 'no releases' })
 
 t.create('Version label should be `no releases` if no pre-release')
-  .get('/vpre/bootstrap.json')
+  .get('/v/bootstrap.json?include_prereleases')
   .intercept(nock =>
     nock('https://libraries.io')
       .get('/api/bower/bootstrap')
       .reply(200, { normalized_licenses: [], latest_release_number: null })
   )
   .expectBadge({ label: 'bower', message: 'no releases' })
+
+t.create('Version (legacy redirect: vpre)')
```

Comment:
Instead of testing `.json` resource we can test `/vpre/bootstrap.svg` or `/vpre/bootstrap` (because we are verifying a header, not a JSON content) - both are redirecting to `/bower/v/bootstrap.svg?include_prereleases`. Properly working redirects for SVG badges are more important than JSON resources.

Reply:
Agreed - I've changed it for the services already touched in this PR.
I'll defer doing the same on other services to another PR.